layouts/_default/post.html

```diff
@@ -7,8 +7,9 @@
       <li class="list-inline-item mr-3 ml-0"><i class="ti-calendar"></i> {{ .PublishDate.Format "02 Jan, 2006" }}</li>
       <!-- author -->
       <li class="list-inline-item mr-3 ml-0"><i class="ti-user"></i> 
-        {{ range split .Params.Author "," }}
-          <a href="{{ `team/` | relLangURL }}{{ . | urlize }}">{{ . | title }}</a>
+        {{ range split .Params.Author ", " }}
+          {{$post := (index (where (where site.RegularPages "Type" "team") "Title" . ) 0).RelPermalink}}
```

Comment:
Do you want to do a check if a team member exists or is this on the basis that all authors of the posts are members that are within Dataworkz?
Thinking of guest writers, leavers, those kind of peoples

Reply:
That actually works out fine, see e.g. Post by leslaw-kubosz. It links the non-exisiting team member to the current page and not a 404 of a missing page. The general problem was that for the blog post backlinking to the author, it took the Author and urlized it for the team member. The other way, going from the Team page, listing all Blog posts of a member, it went by checking the Title Parameter of the member against the authors. That made it an hidden requirement that the team member page has the same Title and url for both to work.

Reply:
Ah, I see what you mean. For the single post it works(it stays on the page) but for the overview the screen reset to the top. Imo a good solution for now, the best solution could be to not have to link at all if its not needed. Like I said, imo.